frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -279,6 +279,9 @@ export const NamespacedClusterServiceVersionTableRow = withFallback<
   const route = resourceObjPath(obj, referenceFor(obj));
   const uid = getUID(obj);
   const internalObjects = getInternalObjects(obj);
+  const getProvidedAPIs = providedAPIsFor(obj).filter(
```

Comment:
This makes the var sound like it's a function. I'd call this `providedAPIs`

```suggestion
  const providedAPIs = providedAPIsFor(obj).filter(
```

Reply:
> This makes the var sound like it's a function. I'd call this `providedAPIs`

@spadgett Done, thanks.